# Project 3

## Has the Trump Administration’s Return to Power Decreased International Air Passenger to the U.S.?

By Abi Insani

### Introduction

**Dataset:**

I am using two datasets:
- [**Bureau of Transport Statistics (BTS): T-100 International Markets in 2024 and 2025**](https://www.transtats.bts.gov/Fields.asp?gnoyr_VQ=FJD) (# of Passengers enter U.S.)

This dataset focuses the change in number of travelers into U.S.A that includes Domestic and International Air Carriers


**Research Question**

*After the Trump Administration returned to office in 2025, has the number of international air travelers to the United States decreased due to the administration’s stricter visa policies and enhanced enforcement measures?*



Columns that will (likely) be used:

From T-100:

- "PASSENGERS"

- "ORIGIN_COUNTRY" / "ORIGIN_COUNTRY_NAME"

- "DEST_COUNTRY" / "DEST_COUNTRY_NAME"

- "YEAR"

- "MONTH"

**Hypothesis**

*H₁: International air travel to the United States decreased comparing year-on-year 2024 vs 2025 data, following the Trump Administration’s implementation of stricter visa rules and enforcement policies.*


### I. Data Sources and Preparation

This project examines whether international travel to the United States declined after the Trump Administration’s stricter visa enforcement began on January 20, 2025. To evaluate this, I compile monthly international arrivals data from Bureau of Transport Statistics between January 1 2024 and October 1 2025.

*Datasets Used:*

[**Bureau of Transport Statistics (BTS): T-100 International Markets in 2024 and 2025**](https://www.transtats.bts.gov/DL_SelectFields.aspx?gnoyr_VQ=FJD&QO_fu146_anzr=Nv4%20Pn44vr45)

**Preparation Steps:**

1. Import U.S. international airport arrivals by BTS from January 1 2024 to October 1 2025 (monthly data).

2. Merge two column (Month and Year) into one and concate both dataset.

3. Show only United States as Destination Country.

Lets begin! First I need to import the necessary packages - pandas and plotly to analyze the data and display charts and other visualization tools

In [1]:
import pandas as pd
import plotly.express as px
import plotly.io as pio

pio.renderers.default = "notebook_connected+plotly_mimetype"

### Step 1: Import U.S. international airport arrivals by BTS (using Monthly Data)

Now we read the International Arrivals in 2024 as **arr_2024**, convert to dataframe as **df_2024**, and observe the Data Types:

In [2]:
# Read CSV data in 2024
arr_2024 = pd.read_csv("T_T100I_MARKET_ALL_CARRIER_2024.csv")
arr_2024.head()
df_2024 = pd.DataFrame(arr_2024)
print(df_2024.dtypes)
df_2024.head(1000)

PASSENGERS             float64
UNIQUE_CARRIER          object
UNIQUE_CARRIER_NAME     object
CARRIER                 object
CARRIER_NAME            object
ORIGIN_AIRPORT_ID        int64
ORIGIN                  object
ORIGIN_CITY_NAME        object
ORIGIN_COUNTRY          object
ORIGIN_COUNTRY_NAME     object
DEST_AIRPORT_ID          int64
DEST                    object
DEST_CITY_NAME          object
DEST_COUNTRY            object
DEST_COUNTRY_NAME       object
YEAR                     int64
MONTH                    int64
dtype: object


PASSENGERS UNIQUE_CARRIER  \
0           0.0            09Q   
1           0.0            09Q   
2           0.0            09Q   
3           0.0            09Q   
4           0.0            09Q   
..          ...            ...   
995         0.0             5X   
996         0.0             5X   
997         0.0             5X   
998         0.0             5X   
999         0.0             5X   

                                   UNIQUE_CARRIER_NAME CARRIER  \
0    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
1    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
2    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
3    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
4    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
..                                                 ...     ...   
995                              United Parcel Service      5X   
996                              United Parcel Service      5X   
997                              United Parcel Service      5X   
998                              United Parcel Service      5X   
999                              United Parcel Service      5X   

                                          CARRIER_NAME  ORIGIN_AIRPORT_ID  \
0    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              10185   
1    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              10540   
2    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
3    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
4    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
..                                                 ...                ...   
995                              United Parcel Service              14100   
996                              United Parcel Service              14100   
997                              United Parcel Service              14100   
998                              United Parcel Service              14100   
999                              United Parcel Service              14100   

    ORIGIN  ORIGIN_CITY_NAME ORIGIN_COUNTRY ORIGIN_COUNTRY_NAME  \
0      AEX    Alexandria, LA             US       United States   
1      BED       Bedford, MA             US       United States   
2      CVG    Cincinnati, OH             US       United States   
3      CVG    Cincinnati, OH             US       United States   
4      CVG    Cincinnati, OH             US       United States   
..     ...               ...            ...                 ...   
995    PHL  Philadelphia, PA             US       United States   
996    PHL  Philadelphia, PA             US       United States   
997    PHL  Philadelphia, PA             US       United States   
998    PHL  Philadelphia, PA             US       United States   
999    PHL  Philadelphia, PA             US       United States   

     DEST_AIRPORT_ID DEST         DEST_CITY_NAME DEST_COUNTRY  \
0              13306  MID         Merida, Mexico           MX   
1              16083  YHU       Montreal, Canada           CA   
2              14286  PTY    Panama City, Panama           PA   
3              14286  PTY    Panama City, Panama           PA   
4              16229  YVR      Vancouver, Canada           CA   
..               ...  ...                    ...          ...   
995            11548  EMA  Derby, United Kingdom           GB   
996            11548  EMA  Derby, United Kingdom           GB   
997            11548  EMA  Derby, United Kingdom           GB   
998            11548  EMA  Derby, United Kingdom           GB   
999            11548  EMA  Derby, United Kingdom           GB   

    DEST_COUNTRY_NAME  YEAR  MONTH  
0              Mexico  2024      1  
1              Canada  2024      2  
2              Panama  2024      1  
3              Panama  2024      3  
4              Canada  2024      2  
..                ...   ...    ...  
995    United Kingdom  2024      

Now we read the International Arrivals in 2024 as **arr_2025**, convert to dataframe as **df_2025**, and observe the Data Types:

In [3]:
# Read CSV data in 2025
arr_2025 = pd.read_csv("T_T100I_MARKET_ALL_CARRIER_2025.csv")
arr_2025.head()
df_2025 = pd.DataFrame(arr_2025)
print(df_2025.dtypes)
df_2025.head(1000)

PASSENGERS             float64
UNIQUE_CARRIER          object
UNIQUE_CARRIER_NAME     object
CARRIER                 object
CARRIER_NAME            object
ORIGIN_AIRPORT_ID        int64
ORIGIN                  object
ORIGIN_CITY_NAME        object
ORIGIN_COUNTRY          object
ORIGIN_COUNTRY_NAME     object
DEST_AIRPORT_ID          int64
DEST                    object
DEST_CITY_NAME          object
DEST_COUNTRY            object
DEST_COUNTRY_NAME       object
YEAR                     int64
MONTH                    int64
dtype: object


PASSENGERS UNIQUE_CARRIER                          UNIQUE_CARRIER_NAME  \
0           0.0            1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC   
1           0.0            1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC   
2           0.0            1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC   
3           0.0            1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC   
4           0.0            1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC   
..          ...            ...                                          ...   
995         0.0             5Y                               Atlas Air Inc.   
996         0.0             5Y                               Atlas Air Inc.   
997         0.0             5Y                               Atlas Air Inc.   
998         0.0             5Y                               Atlas Air Inc.   
999         0.0             5Y                               Atlas Air Inc.   

    CARRIER                                 CARRIER_NAME  ORIGIN_AIRPORT_ID  \
0       1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC              10185   
1       1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC              10194   
2       1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC              10194   
3       1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC              10257   
4       1BQ  Eastern Airlines f/k/a Dynamic Airways, LLC              12206   
..      ...                                          ...                ...   
995      5Y                               Atlas Air Inc.              10299   
996      5Y                               Atlas Air Inc.              10299   
997      5Y                               Atlas Air Inc.              10299   
998      5Y                               Atlas Air Inc.              10299   
999      5Y                               Atlas Air Inc.              10299   

    ORIGIN          ORIGIN_CITY_NAME ORIGIN_COUNTRY ORIGIN_COUNTRY_NAME  \
0      AEX            Alexandria, LA             US       United States   
1      AFW     Dallas/Fort Worth, TX             US       United States   
2      AFW     Dallas/Fort Worth, TX             US       United States   
3      ALB                Albany, NY             US       United States   
4      HRL  Harlingen/San Benito, TX             US       United States   
..     ...                       ...            ...                 ...   
995    ANC             Anchorage, AK             US       United States   
996    ANC             Anchorage, AK             US       United States   
997    ANC             Anchorage, AK             US       United States   
998    ANC             Anchorage, AK             US       United States   
999    ANC             Anchorage, AK             US       United States   

     DEST_AIRPORT_ID DEST   DEST_CITY_NAME DEST_COUNTRY DEST_COUNTRY_NAME  \
0              16149  YOW   Ottawa, Canada           CA            Canada   
1              12911  LCA  Larnaca, Cyprus           CY            Cyprus   
2              16271  YYZ  Toronto, Canada           CA            Canada   
3              16149  YOW   Ottawa, Canada           CA            Canada   
4              14664  RZE  Rzeszow, Poland           PL            Poland   
..               ...  ...              ...          ...               ...   
995            13658  NGB    Ningbo, China           CN             China   
996            13658  NGB    Ningbo, China           CN             China   
997            13658  NGB    Ningbo, China           CN             China   
998            13692  NKG   Nanjing, China           CN             China   
999            13692  NKG   Nanjing, China           CN             China   

     YEAR  MONTH  
0    2025      6  
1    2025      6  
2    2025      2  
3    2025      6  
4    2025      8  
..    ...    ...  
995  2025      6  
996  2025      7  
997  2025      8  
998  2025      1  
999  2025      2  

[1000 rows x 17 columns]

### Step 2: Merge two column (Month and Year) into one and concate both dataset

Since the date/time datasets are Month and Year in separate columns, we want to merge them into one column (Month, Year) and merge dataset into one as **df**.


In [4]:
# Merge Column "YEAR" and "MONTH" into one (2024 and 2025 dataset) SOURCE: CHATGPT

# 2024 Data
df_2024["DATE"] = pd.to_datetime(
    df_2024["YEAR"].astype(int).astype(str)
    + "-"
    + df_2024["MONTH"].astype(int).astype(str).str.zfill(2),
    format="%Y-%m",
)

# 2025 Data
df_2025["DATE"] = pd.to_datetime(
    df_2025["YEAR"].astype(int).astype(str)
    + "-"
    + df_2025["MONTH"].astype(int).astype(str).str.zfill(2),
    format="%Y-%m",
)

# Concate both dataset
df = pd.concat([df_2024, df_2025], ignore_index=True)
print(df.dtypes)
df.head(1000)


PASSENGERS                    float64
UNIQUE_CARRIER                 object
UNIQUE_CARRIER_NAME            object
CARRIER                        object
CARRIER_NAME                   object
ORIGIN_AIRPORT_ID               int64
ORIGIN                         object
ORIGIN_CITY_NAME               object
ORIGIN_COUNTRY                 object
ORIGIN_COUNTRY_NAME            object
DEST_AIRPORT_ID                 int64
DEST                           object
DEST_CITY_NAME                 object
DEST_COUNTRY                   object
DEST_COUNTRY_NAME              object
YEAR                            int64
MONTH                           int64
DATE                   datetime64[ns]
dtype: object


PASSENGERS UNIQUE_CARRIER  \
0           0.0            09Q   
1           0.0            09Q   
2           0.0            09Q   
3           0.0            09Q   
4           0.0            09Q   
..          ...            ...   
995         0.0             5X   
996         0.0             5X   
997         0.0             5X   
998         0.0             5X   
999         0.0             5X   

                                   UNIQUE_CARRIER_NAME CARRIER  \
0    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
1    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
2    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
3    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
4    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...     09Q   
..                                                 ...     ...   
995                              United Parcel Service      5X   
996                              United Parcel Service      5X   
997                              United Parcel Service      5X   
998                              United Parcel Service      5X   
999                              United Parcel Service      5X   

                                          CARRIER_NAME  ORIGIN_AIRPORT_ID  \
0    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              10185   
1    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              10540   
2    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
3    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
4    Swift Air, LLC d/b/a Eastern Air Lines d/b/a E...              11193   
..                                                 ...                ...   
995                              United Parcel Service              14100   
996                              United Parcel Service              14100   
997                              United Parcel Service              14100   
998                              United Parcel Service              14100   
999                              United Parcel Service              14100   

    ORIGIN  ORIGIN_CITY_NAME ORIGIN_COUNTRY ORIGIN_COUNTRY_NAME  \
0      AEX    Alexandria, LA             US       United States   
1      BED       Bedford, MA             US       United States   
2      CVG    Cincinnati, OH             US       United States   
3      CVG    Cincinnati, OH             US       United States   
4      CVG    Cincinnati, OH             US       United States   
..     ...               ...            ...                 ...   
995    PHL  Philadelphia, PA             US       United States   
996    PHL  Philadelphia, PA             US       United States   
997    PHL  Philadelphia, PA             US       United States   
998    PHL  Philadelphia, PA             US       United States   
999    PHL  Philadelphia, PA             US       United States   

     DEST_AIRPORT_ID DEST         DEST_CITY_NAME DEST_COUNTRY  \
0              13306  MID         Merida, Mexico           MX   
1              16083  YHU       Montreal, Canada           CA   
2              14286  PTY    Panama City, Panama           PA   
3              14286  PTY    Panama City, Panama           PA   
4              16229  YVR      Vancouver, Canada           CA   
..               ...  ...                    ...          ...   
995            11548  EMA  Derby, United Kingdom           GB   
996            11548  EMA  Derby, United Kingdom           GB   
997            11548  EMA  Derby, United Kingdom           GB   
998            11548  EMA  Derby, United Kingdom           GB   
999            11548  EMA  Derby, United Kingdom           GB   

    DEST_COUNTRY_NAME  YEAR  MONTH       DATE  
0              Mexico  2024      1 2024-01-01  
1              Canada  2024      2 2024-02-01  
2              Panama  2024      1 2024-01-01  
3              Panama  2024      3 2024-03-01  
4              Canada  2024      2 2024-02-01  
..  

### Step 3: Show only United States as Destination Country

To determine ALL incoming international US-bound flights, we can filter the merged dataset when the **DESTINATION_COUNTRY = US**

In [ ]:
# Filter Destination Country = United States SOURCE: CHATGPT
df_DEST_US = df[df["DEST_COUNTRY"] == "US"]

df_DEST_US.head()

df_monthly = (
    df_DEST_US.groupby("DATE", as_index=False)["PASSENGERS"].sum().sort_values("DATE")
)

df_monthly = df_monthly[
    (df_monthly["DATE"] >= "2024-01-01") & (df_monthly["DATE"] <= "2025-07-01")
]

### II. Data Visualization and Analysis

To visualize monthly changes in US-bound international air traffic, we construct a line chart of total inbound passengers from January 2024 to October 2025. This visualization highlights the month-to-month flow of international arrivals and allows a clear comparison between the same months in 2024 and 2025, focusing especially on January–July.

In this step, we derive a Year-on-Year (YoY) table for January–July by aggregating monthly totals from both years and calculating the percentage change for each corresponding month. This YoY metric helps identify emerging trends in inbound passenger demand across the two periods.

Steps

1. Plot Monthly Date vs Total Passengers for All US Inbound Flights.

2. Filter January–July for both years and compute month-specific totals for 2024 and 2025.

3. Interpret the results.

### Step 1: Plot Monthly Date vs Total Passengers for All US Inbound Flights

To estimate the monthly rate of US-bound international passengers, we plot the data using a monthly timeline from January to July for both 2024 and 2025. This allows a direct comparison of passenger volumes across the same period in each year.

In this step, we also calculate the year-on-year (YoY) change for each month (January–July), showing how 2025 passenger levels differ from those in 2024:

In [ ]:
# Plot the Line Chart
import plotly.express as px

fig = px.line(
    df_monthly,
    x="DATE",
    y="PASSENGERS",
    title="Monthly Passenger Volume (Jan 2024 – July 2025)",
    markers=True,
)

fig.update_layout(
    xaxis_title="Month", yaxis_title="Total Passengers", hovermode="x unified"
)

fig.show()


**Observations**

The monthly passenger volume from January 2024 through July 2025 shows common clear patterns, with peaks usually occurs during mid-summer months and lower demand in late winter and early spring. Passenger traffic decreased sharply in February 2024 and February 2025, followed by a strong rebound each March and a and summer surge from May to July, reaching the highest levels of over 13 million travelers in July 2024 and again in July 2025. Overall, the trend suggests seasonality in international air travel, with rising summer demand and modest year-over-year growth entering mid-2025.

*Note that 2025 data only extends through July, as BTS has not yet released second-semester 2025 passenger figures.*

### Step 2: Filter Month from January-July both in 2024 and 2025 and Calculate the YoY Difference

We want to show only data from month January to July for 2024 and 2025 period. Afterwards, we prepare the table for YoY data comparison.

In [9]:
# Filter Month from January-July both in 2024 and 2025 and Pivot the Table SOURCE: CHATGPT
df_yoy = (
    df_monthly[df_monthly["DATE"].dt.month <= 7]
    .assign(YEAR=lambda x: x["DATE"].dt.year, MONTH=lambda x: x["DATE"].dt.month)
    .pivot_table(index="MONTH", columns="YEAR", values="PASSENGERS", aggfunc="sum")
    .assign(
        **{"Year-on-Year Change (%)": lambda x: (x[2025] - x[2024]) / x[2024] * 100}
    )
    .reset_index()[["MONTH", 2024, 2025, "Year-on-Year Change (%)"]]
)

print(df_yoy)

YEAR  MONTH        2024        2025  Year-on-Year Change (%)
0         1  10263112.0  10661430.0                 3.881065
1         2   8801805.0   8712879.0                -1.010315
2         3  10851474.0  10781455.0                -0.645249
3         4  10520585.0  10770295.0                 2.373537
4         5  10643150.0  10498794.0                -1.356328
5         6  11665584.0  11562102.0                -0.887071
6         7  13252962.0  12996460.0                -1.935431


### Step 3: Interpret the Results

The year-on-year comparison of US-bound international passengers from January to July shows a mixed pattern across the first semester. Passenger traffic grew by 3.88% in January, adding more than 300,000 travelers compared to January 2024. However, most subsequent months recorded slight declines, with the exception of April, which saw a 2.37% increase in passenger volume. 

Overall, the first-half year-on-year trend suggests an overall decline in US-bound international air travel when comparing 2025 to 2024 for the January–July period.

**Explanations**

Based on the Trump administration policies of tightening visa and immigration rules such as:
1. Stricter non-immigration visa screening
2. Reduced visa issuance to students, tourists, and work visas
3. Expansion of travel restrictions
4. Donald Trump rhetoric of anti-immigrations

It is also important to note that this dataset captures only air travel, not total international entries. Broader migration flows through land borders and seaports are not reflected here, which limits the completeness of the analysis.

In addition, due to the limitation of data in 2025, we are not able to make definite conclusion to prove our hypothesis. After we collect the remaining 2025, more establised finding can be achieved.

**Conclusion**

Overall, the early evidence suggests a potential decline in international air travel to the United States in 2025, but final conclusions should be drawn only after complete data becomes available. So we cannot prove our initial hypothesis that International air travel to the United States decreased comparing year-on-year 2024 vs 2025 data due to Trump Administration’s implementation of stricter visa rules and enforcement policies.
